src/components/activities.js

```diff
@@ -13,21 +13,22 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import activityPorts from 'web-activities/activity-ports';
+
 import {
   AnalyticsRequest,
   EventOriginator,
   deserialize,
   getLabel,
 } from '../proto/api_messages';
-import {INTERNAL_RUNTIME_VERSION} from '../constants';
-
 import {Constants} from '../utils/constants';
+import {INTERNAL_RUNTIME_VERSION} from '../constants';
 import {addQueryParam} from '../utils/url';
 
 const {
   ActivityIframePort: WebActivityIframePort,
   ActivityPorts: WebActivityPorts,
-} = require('web-activities/activity-ports');
```

Comment:
Isn't this change not required with the commonjs plugin?

Reply:
Good point, fixed!